packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -109,10 +114,15 @@ const BlockToolbar = ( { hideDragHandle } ) => {
 
 	const shouldShowVisualToolbar = isValid && isVisual;
 	const isMultiToolbar = blockClientIds.length > 1;
+	const isSynced =
+		isReusableBlock( blockType ) || isTemplatePart( blockType );
 
 	const classes = classnames(
 		'block-editor-block-toolbar',
-		shouldShowMovers && 'is-showing-movers'
+		shouldShowMovers && 'is-showing-movers',
+		{
+			'is-synced': isSynced,
+		}
```

Comment:
Nit: Should we move the `is-showing-movers` class into the object so it's consistent?

Reply:
Good point, have updated.